### PR TITLE
Bump version to 1.14

### PR DESCRIPTION
**Pull request by irvifa**
Bump version to 1.14

### CODE REVIEW

File: kubernetes/kubernetes-client-core.cabal [hunk 1]
```diff
@@ -8,16 +8,16 @@ description:    .
...
-homepage:       https://github.com/kubernetes-client/haskell
-author:         Auto Generated
-maintainer:     Shimin Guo <smguo2001@gmail.com>
-license:        Apache-2.0
-license-file:   LICENSE
+homepage:       https://openapi-generator.tech
+author:         Author Name Here
+maintainer:     author.name@email.com
+copyright:      YEAR - AUTHOR
+license:        UnspecifiedLicense
```
Comment (guoshimin) [inline kubernetes/kubernetes-client-core.cabal]: Are you using the latest master version of https://github.com/kubernetes-client/gen? It should not have changed these fields.

Comment (irvifa) [reply kubernetes/kubernetes-client-core.cabal]: Yes, i'm using the latest version

Comment (guoshimin) [reply kubernetes/kubernetes-client-core.cabal]: What was the command you used to generate the code?

We override these fields here: https://github.com/kubernetes-client/gen/blob/3e160e6416503e9983c0ff2ff4f05e4455bc393c/openapi/haskell.sh#L55-L69

Comment (irvifa) [reply kubernetes/kubernetes-client-core.cabal]: I'm using `./$(GEN_ROOT)/openapi/haskell.sh kubernetes settings`

Comment (irvifa) [reply kubernetes/kubernetes-client-core.cabal]:
```
CABAL_OVERRIDES=(homepage https://github.com/kubernetes-client/haskell
           author "Auto Generated"
           maintainer "Shimin Guo <smguo2001@gmail.com>"
           license Apache-2.0)

patch_cabal_file() {
    while [[ $# -gt 1 ]]; do
        sed -i 's|^\('$1':[[:space:]]*\).*|\1'"$2"'|' ${OUTPUT_DIR}/*.cabal
        shift 2
    done
}
patch_cabal_file "${CABAL_OVERRIDES[@]}"

# Add license-file after license
sed -i '/^license:/a license-file:   LICENSE' ${OUTPUT_DIR}/*.cabal

ln -sf ../LICENSE ${OUTPUT_DIR}/LICENSE

sed -i '/^copyright:/d' ${OUTPUT_DIR}/*.cabal
echo "---Done."

```

Comment (irvifa) [reply kubernetes/kubernetes-client-core.cabal]: @guoshimin I found the error
```
sed: 1: "/Users/irvi.aini/dev/k8 ...": command i expects \ followed by text
```

```
uname -a
jessie 18.2.0 Darwin Kernel Version 18.2.0: Thu Dec 20 20:46:53 PST 2018; root:xnu-4903.241.1~1/RELEASE_X86_64 x86_64 i386 MacBookPro14,1 Darwin
```

Comment (irvifa) [reply kubernetes/kubernetes-client-core.cabal]: FIxed by changing `sed` with `gsed`

**Comment by irvifa** [comment]
Related to https://github.com/kubernetes-client/gen/issues/116

**Comment by guoshimin** [comment]
Sorry for the delay in reviewing. Testing it here: https://travis-ci.org/guoshimin/kubernetes-client-haskell/builds/537368152

**Comment by irvifa** [comment]
@guoshimin is there a way to manage backward compatibility for this, also would like to run the local test on my local machine before pushing any changes, is that possible?

**Comment by guoshimin** [comment]
To run tests locally, you can see the travis config at https://travis-ci.org/guoshimin/kubernetes-client-haskell/builds/537368152/config, or just stack test / cabal test.

For generated code, I don't think we need to do anything about backward compatibility beyond what's offered in the API itself.

Looks like the test failures are not in the generated code, but due to lack of version bounds in the hand-written code. OK to merge.

/lgtm
/approve

**Pull request merged**